bin/stress-test/src/main.rs

```diff
@@ -0,0 +1,287 @@
+
+/// Create and store blocks into the store. Create a given number of accounts, where each account
+/// consumes a note created from a faucet. The cli accepts the following parameters:
+/// - `dump_file`: Path to the store database file.
+/// - `num_accounts`: Number of accounts to create.
+/// - `genesis_file`: Path to the genesis file of the store.
+#[tokio::main]
+async fn main() {
+    let cli = Cli::parse();
+
+    match &cli.command {
+        Command::SeedStore { dump_file, num_accounts, genesis_file } => {
+            seed_store(dump_file, *num_accounts, genesis_file).await;
+        },
+    }
+}
+
+/// Seed the store with a given number of accounts.
+async fn seed_store(dump_file: &Path, num_accounts: usize, genesis_file: &Path) {
+    let store_config = StoreConfig {
+        database_filepath: dump_file.to_path_buf(),
+        genesis_filepath: genesis_file.to_path_buf(),
+        ..Default::default()
+    };
+
+    // Start store
+    let store = Store::init(store_config.clone()).await.context("Loading store").unwrap();
+    task::spawn(async move { store.serve().await.context("Serving store") });
+    let start = Instant::now();
+
+    // Create faucet
+    println!("Creating new faucet account...");
+    let store_client =
+        StoreClient::new(ApiClient::connect(store_config.endpoint.to_string()).await.unwrap());
+    let genesis_header = store_client.latest_header().await.unwrap();
+    let faucet_id = create_faucet(&genesis_header);
+
+    // Spawn first block builder task
+    let (batch_sender, batch_receiver) = unbounded_channel::<TransactionBatch>();
+    let db_task = task::spawn(build_blocks(batch_receiver, store_client));
+
+    // Create notes
+    println!("Creating notes...");
+    let notes = task::spawn_blocking(move || {
+        generate_note_batches(num_accounts, faucet_id, batch_sender.clone())
+    })
+    .await
+    .unwrap();
+
+    let (insertion_time, num_insertions) = db_task.await.unwrap();
+    println!(
+        "Created notes: inserted {} blocks with avg insertion time {} ms",
+        num_insertions,
+        (insertion_time / num_insertions).as_millis()
+    );
+
+    // Spawn second block builder task
+    let store_client =
+        StoreClient::new(ApiClient::connect(store_config.endpoint.to_string()).await.unwrap());
+    let (batch_sender, batch_receiver) = unbounded_channel::<TransactionBatch>();
+    let db_task = task::spawn(build_blocks(batch_receiver, store_client));
+
+    // Create accounts to consume the notes
+    println!("Creating accounts and consuming notes...");
+    task::spawn_blocking(move || {
+        generate_account_batches(num_accounts, &notes, batch_sender, &genesis_header);
+    })
+    .await
+    .unwrap();
+    let (insertion_time, num_insertions) = db_task.await.unwrap();
+    println!(
+        "Consumed notes: inserted {} blocks with avg insertion time {} ms",
+        num_insertions,
+        (insertion_time / num_insertions).as_millis()
+    );
```

Comment:
Would be cool to print out average block insertion time per some number of blocks (i.e., for every 1000 blocks). This way, we'd see how the insertion time changes the the number of blocks in the store increases.

Also, would be cool to print out the store database file size, ideally for each block interval as well.